README.md

```diff
@@ -1,12 +1,10 @@
-# Jenkins Community Kubernetes Helm Charts
+# Jenkins Helm Charts for Long-Term-Support (LTS) releases
```

Comment:
why is this for LTS releases? this is for anything? the helm chart can have weekly features in it, the default image just defaults to LTS.

Reply:
> the default image just defaults to LTS.

My thoughts were to reword to LTS, given the zip we distribute contains LTS versions only. Of course, you can package one for weeklys yourself, but due to lack of documentation I felt LTS is more appropriate.

Reply:
Created https://github.com/jenkinsci/helm-charts/pull/851